web_src/css/repo.css

```diff
@@ -2128,14 +2128,17 @@
 }
 
 #search-user-box .results .result .image {
-  float: left;
-  margin-right: 8px;
+  order: 0;
+  margin-right: 12px;
   width: 2em;
   height: 2em;
+  min-width: 2em;
+  min-height: 2em;
 }
 
 #search-user-box .results .result .content {
   margin: 6px 0; /* this trick is used to align with the sibling avatar image */
+  overflow-wrap: anywhere;
```

Comment:
Is the comment above `/* this trick is used to align with the sibling avatar image */` still correct?

Reply:
No, it's unrelated, without this a unbroken description line would overflow the box:

<img width="272" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/70fc7635-8031-420c-a70c-de331f507be3">

Reply:
This is because fomantic's `.ui.search > .results { width: 18em; }` so I guess I will move this to the same selector.

Reply:
I mean, if the comment is not correct anymore, does it need to be updated?

Reply:
Selector moved.

Reply:
Comment is not accurate anymore, but the rule is still needed to avoid strange rendering, I will just remove the comment.